l1-contracts/test/shouting.t.sol

```diff
@@ -22,7 +22,7 @@ contract ScreamAndShoutTest is Test {
     bytes32 codeHash = keccak256(creationCode);
 
     assertEq(
-      codeHash, 0x8660a25b0d480424b950324ef5d078cfb12a06d5c807ab09ade301968ac20ae1, ERR_STRING
```

Comment:
Was this pointed at master first? Since this should not have yelled at your otherwise 👀 Mainly to figure out if we are correctly running it.